### PR TITLE
r? ianbicking Containerize pageshot with docker and elastic beanstalk

### PR DESCRIPTION
**Pull request by fzzzy**

### CODE REVIEW

File: server/src/server.js [hunk 1]
```diff
@@ -40,6 +40,10 @@ app.use("/static", express.static(path.join(__dirname, "static"), {
   index: false
 }));
 
+let xpidir = path.join(__dirname, "..", "xpi");
+console.log("xpidir", xpidir);
```
Comment (ianb) [inline server/src/server.js]: Oops, console.log

File: server/src/dbschema.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const MAX_DB_LEVEL = 6;
 /** Create all the tables */
 exports.createTables = function () {
   console.info("Setting up tables on", db.constr);
+  console.info("Environment: ", process.env);
```
Comment (ianb) [inline server/src/dbschema.js]: Is there a reason for the whole environment to be in here?  Could potentially lead to leaking private keys into the logs, for instance.

**Comment by ianb** [comment]
Just wondering about those two log messages, otherwise r+/commitable

**Comment by fzzzy** [comment]
Removed debug logging.

**Pull request merged**